macOSUpgrade.sh

```diff
@@ -107,7 +107,7 @@ fi
 ##Full Screen by default
 ##Use Parameter 9 in the JSS.
 userDialog="$9"
-if [[ ${userDialog:=0} != 1 ]]; then userDialog=0 ; fi
+if [[ ${userDialog:=0} -ne 1 ]]; then userDialog=0 ; fi
```

Comment:
You would like here should be `[  ${userDialog:=0} -ne 1 ]`

Reply:
I changed this. https://github.com/kc9wwh/macOSUpgrade/pull/114/commits/7cb102b03bbd9b411a269224a41f2427ebcdf92f